vnext/ReactUWP/Utils/BaseScriptStoreImpl.cpp

```diff
@@ -0,0 +1,341 @@
+#include "pch.h"
+
+#include "BaseScriptStoreImpl.h"
+
+#include <fstream>
+
+#include <winrt/Windows.Storage.FileProperties.h>
```

Comment:
are these headers used?

Reply:
Thanks! We need only Windows.Storage.h and Windows::Storage namespace ..